adapter/parser/src/main/java/org/mobilitydata/gtfsvalidator/parser/GtfsEntityParser.java

```diff
@@ -93,6 +113,20 @@ public GtfsEntityParser(GtfsSpecificationProto.CsvSpecProto fileSchema, RawFileI
                                 )
                         );
                     }
+                } else if (columnSpecProto.getType().getType() ==
+                        GtfsSpecificationProto.ColumnInputType.InputType.DATE) {
+
+                    //FIXME: retrieve locale from agency_lang in agency.txt and if that doesn't exist,
+                    //from feed_lang in feed_info.txt before defaulting to Locale.US
+                    if (!dateValidator.isValid(rawField, DATE_PATTERN, Locale.US)) {
```

Comment:
The good news is that you shouldn't have to handle locales here. The [GTFS spec](https://gtfs.org/reference/static/#field-types) says:

>Service day in the YYYYMMDD format.

So the `, Locale.US` should be removed.